src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentPanelMobile.js

```diff
@@ -50,6 +50,14 @@ class DocumentPanelMobile extends Component {
                     id="DocumentPanelMobile.AfterAuthors"
                     metadata={doc.metadata}
                   />
+                  {doc.metadata.imprint?.publisher
+                    ?
```

Comment:
`condition ? <component> : ""` syntax could be replaced with ` condition && <component>`
same in some code above, please find all the occurences